CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Change Log
 
+## [0.23.0](https://github.com/eclipse/lemminx/milestone/36?closed=1) (December 14, 2022)
+
+### Enhancements
+
+ * Codelens, completion, definition, documentLink, highlighting, references, rename support in RelaxNG file. See [#1400](https://github.com/eclipse/lemminx/pull/1400).
+ * Surround selection with XML element. See [#1389](https://github.com/eclipse/lemminx/pull/1389).
+ * Switch experimental formatter to default. See [#1377](https://github.com/eclipse/lemminx/issues/1377).
```

Comment:
I would call this, "Enable the experimental formatter by default, and retain previous formatter for legacy use."

@datho7561 thoughts ?

Reply:
"Enable the experimental formatter by default and add `xml.format.legacy` setting to retain previous formatter"

Reply:
Yeah that works for me.